scripts/irods_ping.py

```diff
@@ -0,0 +1,59 @@
+from getpass import getpass
```

Comment:
could just use these?

https://docs.irods.org/4.2.8/system_overview/tips_and_tricks/#monitoring-status-of-irods-servers

or 

https://github.com/irods/irods/blob/a4c97f8a65bd8d2b5d7a505612f2d9d670d33957/scripts/irods/controller.py#L103-L113

Reply:
for the ping, yes... I can use the heartbeat. 

There is still the matter of initializing the credentials. The choice there is
   1. install icommands on the client and use `iinit`
   2. use `irods.password_obfuscation` which comes for free with PRC

Reply:
gotcha - yes, PRC is fine fine.   no iinit unless we need it.